11ty/definitions/idiot.md

```diff
@@ -1,8 +1,24 @@
 ---
 title: Idiot
 slug: idiot
-defined: false
+defined: true
+speech: noun
+exerpt: previously a psychiatric term for someone with a profound intellectual disability; typically used as a generic exclamation of frustration at the actions of another person
 flag:
   level: avoid
   text: 'Ableist language'
+reading:
+  - text: 'The Clinical History of "Moron," "Idiot," and Imbecile'
+    href: https://www.merriam-webster.com/words-at-play/moron-idiot-imbecile-offensive-history
 ---
+Since medieval times, the word "idiot" was used to describe an ignorant person, and from the 14th century specifically someone with intellectual and developmental disabilities (I/DD). 
+
+In the early 20th century, the term was adopted by the legal and psychiatric communities to refer to an intellectual disability resulting in function equivalent to a neurotypical two-year old according to contemporary intelligence scales. Like those intelligence measurement systems, the term itself has been replaced in technical settings by more specific medical language.
+
+## Issues
+
+In modern times, "idiot" is rarely used to describe someone with an ID/D, but rather someone who is acting in a manner the speaker finds thoughtless or selfish. Nevertheless, ableist language perpetuates entrenched, dehumanising attitudes toward people with intellectual disabilities.
+
+## Alternatives
+
+Since "idiot" is frequently an exclamation of frustration, use a term that describes the frustrating *action* rather than suggesting an ID/D suffered by the *actor*: "I hated it when that driver cut me off in traffic."
```

Comment:
Since the term is usually an exclamation of frustration, use a term that describes the frustrating *action* rather than insulting the *actor* with an ableist term. 

For example: "I hated it when that driver cut me off in traffic."